pkg/api/v1/config_test.go

```diff
@@ -154,21 +155,22 @@ func TestListDashboardGroups(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		server := setupTestServer(t, &test.config)
-		request, err := http.NewRequest("GET", "/api/v1/dashboardgroups", nil)
-		if err != nil {
-			t.Fatalf("Can't form dashboardgroup request")
-		}
-		response := httptest.NewRecorder()
-		server.ListDashboardGroups(response, request)
-		if response.Code != http.StatusOK {
-			t.Errorf("Expected OK status, but got %v", response.Code)
-		}
-		if response.Body.String() != test.expectedJSON {
-			t.Errorf("In Body, Expected %s; got %s", test.expectedJSON, response.Body.String())
-		}
+		t.Run(test.name, func(t *testing.T) {
+			server := setupTestServer(t, test.config)
+			request, err := http.NewRequest("GET", "/api/v1/dashboardgroups", nil)
+			if err != nil {
+				t.Fatalf("Can't form dashboardgroup request")
+			}
+			response := httptest.NewRecorder()
+			server.ListDashboardGroups(response, request)
+			if response.Code != http.StatusOK {
```

Comment:
Shouldn't we also test non-happy paths?

Reply:
Done. Expanded this test to include passed-in scopes and trying to read an invalid config.